cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -152,8 +154,13 @@ public void execute() {
         String[] args;
         if (this.argList == null) {
             args = new String[0];
+            this.projectPath = Paths.get(System.getProperty("user.dir"));
+        } else if (this.argList.get(0).startsWith(RuntimeConstants.BALLERINA_ARGS_INIT_PREFIX)) {
+            args = argList.toArray(new String[0]);
+            this.projectPath = Paths.get(System.getProperty("user.dir"));
```

Comment:
Can't we use `USER_DIR` in ProjectConstants?

Reply:
Fixed in 96cd790